### ISSUE TITLE
woman => womyn

### ISSUE THREAD

**Issue by pippijn** [JackDanger/feminizer#1]
Please consider using a more appropriate word for your replacements. [Thanks](http://en.wikipedia.org/wiki/Womyn#Reasoning).

**Comment by JackDanger** [comment]
@pippijn This is a very interesting proposal. If this project changes "women" to "men" and changes "men" to "womyn", do you think that's an improvement or confusing?

**Comment by pippijn** [comment]
Good question. This may need more thought than I initially anticipated. Is the feminize function currently its own inverse? If not, then I think it would be an improvement.

**Comment by JackDanger** [comment]
>  Is the feminize function currently its own inverse?

That's correct. The idea is that you can run it twice and get the original output: `feminize(feminize(string)) == string`.

It's not perfect because 'his' and 'him' both translate to 'her' and it's tough to do the reverse but it's otherwise pretty complete.
